jupyterlab_gcedetails/auth_token_server/app.js

```diff
@@ -23,10 +23,10 @@ if (fs.existsSync(keyPath)) {
 const oauth2Client = new google.auth.OAuth2(
   keys.client_id,
   keys.client_secret,
-  keys.redirect_uris[0]
+  'https://jupyterlab-interns-sandbox.uc.r.appspot.com/oauth2callback'
```

Comment:
*nit*, but it's usually a good practice to support local development and production with some sort of logic based on environment variables. So you could something like if NODE_ENV === 'dev' use localhost, otherwise prod.